R/process_lookup_sc_client.R

```diff
@@ -13,6 +13,10 @@
 #' @export
 #' @family process extracts
 process_lookup_sc_client <- function(data, year, write_to_disk = TRUE) {
+  sc_demographics <- read_file(get_sc_demog_lookup_path(),
```

Comment:
The issue with multiple people having the same chi in this file will affect this. This could do with another look once demographics file is sorted out

Reply:
MM - will pick this up again and investigate the multiple people with same chi